src/github_changelog/github.clj

```diff
@@ -54,14 +66,11 @@
         endpoint  (pulls-url config)]
     (throttler/throttle-fn (partial issue-request endpoint) ratelimit :second)))
 
-(defn- get-pulls [config]
+(defn fetch-pulls [config]
   (let [call-api                        (call-api-fn config)
         first-request                   (make-request config)
         first-response                  (call-api first-request)
         {links :links first-body :body} first-response
         rest-requests                   (make-requests config links)
         rest-responses                  (pmap call-api rest-requests)]
     (into first-body (flatten (map :body rest-responses)))))
```

Comment:
```suggestion
    (into first-body (comp (map :body) cat) rest-responses)))))
```
Just putting this on your radar: https://clojuredocs.org/clojure.core/cat
This version is a bit more efficient not only because not allocating an interdemiate `map` lazyseq, but by only flattening on a single level.